### PR TITLE
RESTWS-863: End point test for updating location tag name and descitp…

### PR DESCRIPTION
**Pull request by kdaud**
…tion

## Issue I worked on

see https://issues.openmrs.org/browse/RESTWS-863

## Checklist: I completed these to help reviewers :)

- [x ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [ x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by kdaud** [comment]
@dkayiwa we would add this test as a bonus for REST API. Though its not inclusive on the [Platform 2.4.0 REST-WS endpoint tests](https://docs.google.com/spreadsheets/d/1vrVYIjVZqxzS2FHmD63KGV2wqvpJN-x-k77atS6TtIM/edit#gid=0&range=15:15) but adds value to the REST API in terms of automated testing of the feature in the context.

File: omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/LocationTagController1_8Test.java [hunk 1]
```diff
@@ -109,17 +109,34 @@ public void shouldListAll() throws Exception {
 	}
 	
 	@Test
-	public void shouldUpdateLocationTag() throws Exception {
+	public void shouldUpdateLocationTagName() throws Exception {
 		
-		final String editedName = "Location Tag edited";
-		String json = "{ \"name\":\"" + editedName + "\" }";
+		final String EDITED_NAME = "Location Tag edited";
```
Comment (dkayiwa) [inline omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/LocationTagController1_8Test.java:114]: Why don't you combine name and description update in the same ticket like you have been doing for others?

Comment (kdaud) [reply omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/LocationTagController1_8Test.java:114]: > combine name and description update in the same test

good ideal and makes our code base precise.

**Comment by dkayiwa** [comment]
The above ticket link looks incomplete.

**Comment by kdaud** [comment]
> The above ticket link looks incomplete.

I now agree with the remark from one of your previous comment on a certain pull

> That is the effect of grey hair. relaxed

**Pull request merged**